src/PDS4_LUCY_IngestLDD.xml

```diff
@@ -656,7 +656,7 @@
     <DD_Value_Domain>
       <enumeration_flag>false</enumeration_flag>
       <value_data_type>ASCII_Integer</value_data_type>
-      <unit_of_measure_type>Units_of_Rates</unit_of_measure_type>
+      <unit_of_measure_type>Units_of_Angular_Velocity</unit_of_measure_type>
```

Comment:
The actual unit filled in for this field in the example data products was `"kilobits/s"` (which is indeed a `Units_of_Rates`), but the value in those labels matched the number in the FITS header that reports its unit as `urad/s`. So, I think this change is right, I'm making this note to verify this before merging.

Reply:
D'oh, I forgot that we needed to have urad/s added to the enumerated values of Units of Angular Velocity

Reply:
The other alternative is to do the conversion when my code fills in the label by just doing a multiplication by `1e-6`, but if it's not a pain to add `urad/s` as an allowable unit, that would be great.